spec/support/active_record.rb

```diff
@@ -40,6 +43,26 @@
   end
 end
 
+def create_users_table
+  create_table :users, id: false do |t|
+    t.column :id, :string
+    t.column :name, :string
+  end
+end
+
+ActiveRecord::Base.connection.execute("DROP TABLE IF EXISTS 'countries'")
+ActiveRecord::Base.connection.execute("DROP TABLE IF EXISTS 'cities'")
+ActiveRecord::Base.connection.execute("DROP TABLE IF EXISTS 'locations'")
+ActiveRecord::Base.connection.execute("DROP TABLE IF EXISTS 'comments'")
+ActiveRecord::Base.connection.execute("DROP TABLE IF EXISTS 'users'")
+ActiveRecord::Schema.define do
+  create_countries_table
+  create_cities_table
+  create_locations_table
+  create_comments_table
+  create_users_table
+end
```

Comment:
could elaborate why this is needed? Simply outdated?

Reply:
I needed a new table with `uuid` to test my changes so I created the `users` table. Then having every table creation inside a single block casued Rubocop `Metrics/BlockLength` error. To fix it I moved each table creation to an individual method.

Reply:
you could simply disable rubocop rule, but thanks for slight refactoring 👍